Sources/SwerverCore/Response/ResponseBuilder.swift

```diff
@@ -5,7 +5,9 @@ public class ResponseBuilder {
     private(set) var body: String = String()
     private(set) var headers: [ResponseHeader:String] = [:]
 
-    public init() {}
+    public init() {
+        headers[.contentLength] = String(0)
```

Comment:
This should likely be called during `build`. That way you can incorporate the body if/when it is present.